essm/variables/units.py

```diff
@@ -23,20 +23,46 @@
 import operator
 
 import sympy.physics.units as u
+from sympy import Symbol
 from sympy.physics.units import Dimension, Quantity, find_unit
 from sympy.physics.units.systems import SI
-
+from sympy.physics.units.dimensions import (
+    dimsys_SI, amount_of_substance, luminous_intensity, temperature,
+    frequency, force, pressure, energy, power, charge, voltage,
+    capacitance, conductance, magnetic_flux, magnetic_density,
+    inductance, luminous_intensity, time)
+
+candela = u.candela
+coulomb = u.coulomb
+farad = u.farad
+gray = u.gray
+henry = u.henry
 joule = u.joule
+katal = u.katal
 kelvin = u.kelvin
 kilogram = u.kilogram
+lux = u.lux
 meter = u.meter
-mole = u.mole
+mol = mole = u.mol
+newton = u.newton
+ohm = u.ohm
 pascal = u.pascal
 second = u.second
+siemens = u.siemens
+tesla = u.tesla
+volt = u.volt
 watt = u.watt
+weber = u.weber
 
 SI_DIMENSIONS = {
-    str(Quantity.get_dimensional_expr(d)): d for d in SI._base_units}
+    Quantity.get_dimensional_expr(d): d for d in SI._base_units}
+
+extunits = list(SI._base_units) + [
```

Comment:
can you rename it to something like: `SI_UNITS_EXT` or even better to replace `EXT` with full name.

Reply:
Done, thanks!